src/Okta.Idx.Sdk.E2ETests/Configuration.cs

```diff
@@ -1,14 +1,14 @@
 ﻿namespace Okta.Idx.Sdk.E2ETests
 {
-    public class TestConfig : ITestConfig
+    public class Configuration : IConfiguration
```

Comment:
I like the prefix "Test" on this class.  Is there a technical reason not to name this `TestConfiguration`?